docs/case-studies/gitops.md

```diff
@@ -69,108 +296,113 @@ stringData:
 > 
 > ```shell
 > jobs:
 >  publish:
 >    if: "!contains(github.event.head_commit.message, 'Update image automatically')"
 > ```
 
-Re-check the `Application` in cluster, we can see that the image of the `Application` has been updated after a while. We can `curl` to `Ingress` to see the current version:
+Re-check the `Application` in cluster, we can see that the image of the `Application` has been updated after a while.
 
-```shell 
-$ curl -H "Host:testsvc.example.com" http://<your-ip>
+> KubeVela polls the latest information from the code and image repo periodically (at an interval that can be customized):
+> * When the `Application` file in the Git repository is updated, KubeVela will update the `Application` in the cluster based on the latest configuration.
+> * When a new tag is added to the image registry, KubeVela will filter out the latest tag based on your policy and update it to Git repository. When the files in the repository are updated, KubeVela repeats the first step and updates the files in the cluster, thus achieving automatic deployment.
+
+We can `curl` to `Ingress` to see the current version and data:
+
+```shell
+$ kubectl get ingress
+NAME        CLASS    HOSTS                 ADDRESS         PORTS   AGE
+my-server   <none>   kubevela.example.com  <ingress-ip>    80      162m
+
+$ curl -H "Host:kubevela.example.com" http://<ingress-ip>
 Version: 0.1.6
+
+$ curl -H "Host:kubevela.example.com" http://<ingress-ip>/db
+User: KubeVela
+Description: It's a test user
+
+User: KubeVela2
+Description: It's another test user
 ```
 
 The `Version` has been updated successfully! Now we're done with everything from changing the code to automatically applying to the cluster.
 
-KubeVela polls the latest information from the code and image repo periodically (at an interval that can be customized):
-* When the `Application` file in the Git repository is updated, KubeVela will update the `Application` in the cluster based on the latest configuration.
-* When a new tag is added to the image registry, KubeVela will filter out the latest tag based on your policy and update it to Git repository. When the files in the repository are updated, KubeVela repeats the first step and updates the files in the cluster, thus achieving automatic deployment.
+## Summary
+
+For platform admin/SRE, to update the configuration of the infrastructure (such as database) or other fields of the application, the only thing you need to do is to modify the files in the configuration repository. KubeVela will automatically synchronize the configuration to the cluster, thus simplifying the deployment process.
+
+For end user developers, KubeVela automatically updates the image in the configuration repository after the user modifies the code in the repository, make it easier for developers to deploy and update their apps.
```

Comment:
```suggestion
For end users/developers, they write the source code, push it to Git, and then re-deployment will happen. It will make CI to build the image. KubeVela will then update the image field and apply the deployment config.
```